api-tester/build.gradle

```diff
@@ -6,6 +6,16 @@ plugins {
 apply from: "$rootProject.projectDir/base-application.gradle"
 
 android {
+    flavorDimensions "apis"
+    productFlavors {
+        defaults {
+            dimension "apis"
+        }
+        customEntitlementsComputation {
+            dimension "apis"
+        }
+    }
```

Comment:
Note that I'm adding the flavors to the API tests as well so we can split the api tests by flavor.